src/game-support/gta-5.md

```diff
@@ -5,19 +5,24 @@
     "Grand Theft Auto 5"
 ] -->
 
-{{#template ../templates/rating.md status=Gold installs=Yes opens=Yes}}
+{{#template ../templates/rating.md status=Bronze installs=Yes opens=Yes}}
+
+> [!NOTE]
+> As of 2024, Rockstar Games added BattlEye anti-cheat software. Online play will not work on many official and unofficial servers.
 
 > [!WARNING]
 > The Rockstar Games Launcher will fail with the `The Rockstar Games Launcher failed to initialize!` error when `DXVK` is turned `off` in your `Bottle Configuration`. `DXVK` can turned `on` to workaround the launcher error but you might experience visual gliches (invisible guns, green car windows, brightness and reflection artifacts, etc) if you do so. For a glichless experience we suggest to keep `DXVK` turned `off` and follow the following setup to workaround the launcher issue instead.
 
 ## Setup
 
 - Install in Steam as normal.
+- Add the `-nobattleye` launch option to the game.
```

Comment:
Clarify if this means Steam -> Game -> Properties -> Launch options

or Whisky launch options (I'm assuming Steam, just wanting to make sure)